### ISSUE TITLE
Missing implicit operator [1.14]

### ISSUE THREAD

**Issue by AndreyZM** [bridgedotnet/Bridge#1842]
### Expected

```js
Demo.Programm.class = Demo.Class.op_Implicit(Demo.Programm);
```
### Actual

```js
Demo.Programm.class = Demo.Programm;
```

### Steps To Reproduce
http://deck.net/59ffb4f02fdfd662cf8d6292a4f5d2d2
```csharp
public class Class
{
    private Type type;
    static public implicit operator Class(Type t)
    {
        return new Class(){type = t};
    }
    
    static public implicit operator Type(Class t)
    {
        return t.type;
    }
}

public class Programm
{
    static public Class @class;
    
    static public void Main()
    {
        object t;
        //Valid
        @class = (Class)typeof(Programm);
        t = @class;
        Console.WriteLine(t.GetType() == typeof(Class));//true
        
        //Ivalid, missing Implicit operator
        @class = typeof(Programm);
        t = @class;
        Console.WriteLine(t.GetType() == typeof(Class));//false, but true Expected 
    }
}
```

**Comment by LeonidVeriga** [comment]
Reproducible in 1.14.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and included in the Bridge 15.0.2 release that was pushed to NuGet. 

It will also be included in the upcoming Bridge 15.1.0 release. 

Your original Deck now compiles and runs correctly. 

Hope this helps.
